src/main/java/appeng/core/localization/ButtonToolTips.java

```diff
@@ -137,7 +137,11 @@
 	SchedulingMode,
 	SchedulingModeDefault,
 	SchedulingModeRoundRobin,
-	SchedulingModeRandom;
+	SchedulingModeRandom,
+
+	FilterMode,
+	FilterModeKeep,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)